src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int maxNumber = 100;

```

Comment:
The constant 'maxNumber' should be named with all uppercase letters, with words separated by underscores, to comply with the naming convention for constants. For example, 'MAX_NUMBER'.